documentation/tut/toy-example-from-scratch.rst

```diff
@@ -322,5 +329,27 @@ We can also look at the charging schedule in the `FlexMeasures UI <http://localh
 
 Recall that we only asked for a 12 hour schedule here. We started our schedule *after* the high price peak (at 5am) and it also had to end *before* the second price peak fully realised (at 9pm). Our scheduler didn't have many opportunities to optimize, but it found some. For instance, it does buy at the lowest price (around 3pm) and sells it off when prices start rising again (around 6pm).
 
+.. note:: In case we want to delete toy-account and add it again to run the tutorial, it is recommended to delete the database ``flexmeasures-db`` that we have created for this tutorial because on the same database we will get different IDs for the sensors. Follow the following commands.
+
+.. code-block:: console
```

Comment:
This code block appears outside of the note (you can see that if you run `make update-docs`).

Maybe it should become its own section "Re-run the tutorial" and not a note. And it goes to the very end.

Slight fixing of wording/grammar:

"In case you want to run the tutorial again, deleting and re-creating the toy-account might not suffice, as you then will get different IDs for the sensors than this tutorial shows. It is recommended to delete the database ``flexmeasures-db`` (which we have created for this tutorial). Run the following commands:"